docset/winserver2022-ps/windowsupdate/Get-WindowsUpdateLog.md

```diff
@@ -145,8 +159,9 @@ Accept wildcard characters: False
 ```
 
 ### -ProcessingType
-Specifies the file type that **Get-WindowsUpdateLog** uses for temporary files that are created during intermediate processing.
-The acceptable values for this parameter are:
+
+Specifies the file type that **Get-WindowsUpdateLog** uses for temporary files that are created
+during intermediate processing. The acceptable values for this parameter are:
 
 - CSV (comma-separated values)
```

Comment:
You could also wrap these values in backticks, like `` `CSV` ``, because they're specific values to pass at the command line.

Reply:
backticks have been added and I went through and ensured the same formatting was used for all extension and file names in the doc